packages/nimble-components/src/table/specs/table-keyboard-navigation-hld.md

```diff
@@ -0,0 +1,206 @@
+# Table Keyboard Navigation HLD
+
+## Problem Statement
+
+It is desirable for the table to support navigating and interacting with its contents with the keyboard alone. ARIA provides guidance on how to best accomplish this.
+
+## Links To Relevant Work Items and Reference Material
```

Comment:
Do we have visual designs for a row being focused and a cell being focused? If so it'd be good to link to them here. If not we should put it on Brandon's radar.

Reply:
As mentioned on Teams, I can't load any of the old Adobe XD documents anymore (I always get a password error), so I'm not sure if there's any older designs for it. For prototyping I've been using the standard green border color. We can discuss it on Monday probably.

Reply:
Current plan is to demo the current prototype to Brandon at one of the next team meetings (which uses a 2px border plus our standard border color around the active row/cell), and see if he has feedback.